### DISCUSSION TITLE
About SEI current densitty of ec_reaction_limited.py

### DISCUSSION THREAD

**Discussion by RuiheLi** [pybamm-team/PyBaMM#1535] [Q&A]
In pybamm/models/submodels/interface/sei/ec_reaction_limited.py, Line 73, we have #  j_sei = - C_sei_ec * c_ec * exp(). The expression of C_sei_ec is in pybamm/parameters/lithium_ion_parameters.py, which contains self.U_n_ref. And this U_n_ref is defined in Line 528 in lithium_ion_parameters.py as the Reference OCP based on initial concentration at current collector/electrode interface. I wonder this term is not defined Equation-10 in Yang’s paper (X. G. Yang, J. Power Sources, 360, 28–40 (2017) http://dx.doi.org/10.1016/j.jpowsour.2017.05.110.)

**Comment by valentinsulzer** [comment]
I believe it's from the non-dimensionalization, but we could definitely do with double-checking and writing down the full non-dimensionalization somewhere. Also, parameter values between the different SEI models need to be unified (the same parameter is called different things in different submodels)

**Reply by RuiheLi** [reply]
Thanks so much for the reply!!!
